### ISSUE TITLE
Unable to Login

### ISSUE THREAD

**Issue by maltfield** [wireapp/wire-android#3678]
**Describe the bug**

Last week I updated Wire from v3.78.13 to v3.8.23, and (after install), it prompted me for my credentials. I entered them (copied & pasted from password manager, so I'm sure they're correct), and it gave me the error:

```
Invalid information

PLEASE VERIFY YOUR DETAILS AND TRY AGAIN
```

I changed my email address and I changed my password. I confirmed that these updated credentials work fine on my desktop version of Wire.

I then tried the updated credentials (again, copy & paste so no human error), and the Wire app still said my credentials were incorrect.

I deleted all my Wire cache & data and tried to login again. Same issue.

I tried to install the Wire version from F-droid. This failed to install with a different issue, see #3679

**Comment by maltfield** [comment]
This appears to be a duplicate of #1498

**Comment by maltfield** [comment]
I resolved my issue #3679 and was able to install the version of Wire from F-Droid. I confirmed that I get this same error when attempting to login on the newest version in F-Droid = v3.80.23

```
Invalid information

PLEASE VERIFY YOUR DETAILS AND TRY AGAIN
```

**Comment by maltfield** [comment]
Success! I downgraded Wire to v3.77.10 on F-Droid and I was able to login!

**Comment by maltfield** [comment]
I then did an upgrade of Wire to the latest version available in F-Droid at the time of writing: v3.80.23. When I force-quit Wire and re-opened it, I was still logged-in and everything appears to be working
